src/modules/form/types.ts

```diff
@@ -59,7 +59,6 @@ export interface DynamicFieldProps {
 
 export interface DynamicViewFieldProps {
   item: FormItem;
-  nestingLevel: number;
```

Comment:
Are we _sure_ this isn't used by any child items in the viewing context? I'm not sure if we should remove this, it might be necessary to know later if we do want the appearance to depend on the level of nesting. It's probably better to make the appearance declarative, as suggested here https://github.com/open-path/Green-River/issues/5759, but we haven't really made that jump yet.

Reply:
`QuestionGroup` looks at `nestingLevel` now so I think this change might be adjusting that behavior. Feels out of scope for a form builder change

Reply:
Oh, nevermind, it looks like I guess we _did_ make that jump, since it looks like nesting level isn't being passed for forms anymore. So this seems okay. I must have forgotten.

Reply:
Yes, I just removed it to reduce confusion for now, but I think we removed this entirely on individual items when we were doing updates to improve the mobile experience. It is still used by form groups, I think. Seems like something we could revisit with design?